docs/networking/clusternetwork.md

```diff
@@ -44,14 +44,14 @@ Please refer to [network part](./harvester-network.md) for more details about ne
 
 ### Built-in Cluster Network
 
-Harvester provides a built-in cluster network called `mgmt`. It's different from the custom cluster network. `mgmt`:
+Harvester provides a built-in cluster network called `mgmt`. It's different from the custom cluster network. The mgmt cluster network:
```

Comment:
`mgmt` in L#41 diffs with L#37 `management`, please unifiy them

Reply:
No, mgmt cluster network is different from management network. The management network is a network while mgmt cluster network is a cluster network.